Build Failing

Please fix the build.

https://travis-ci.org/reflexdemon/problem-generateor/jobs/577515304#L2153-L2159

```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   EncryptionServiceTest.encrypt:23 Needs to match String expected:<[7E89B6D66B7B001C7AE6955F122530D9]> but was:<[DD620EF120B7A3121C4C514AA21F9178]>
[ERROR] Errors: 
[ERROR]   EncryptionServiceTest.decrypt:30 » BadPadding Given final block not properly p...
[INFO] 
[ERROR] Tests run: 47, Failures: 1, Errors: 1, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

### Hint:
Change the assertion to decrypt and see if it matches the clear text password.
#### Location:
https://github.com/reflexdemon/problem-generateor/blob/master/src/test/java/io/vpv/math/problemgenerateor/service/EncryptionServiceTest.java#L23

Please remember this application has 100% code coverage. I don't want that to be broken.

--- a/src/test/java/io/vpv/math/problemgenerateor/service/EncryptionServiceTest.java
+++ b/src/test/java/io/vpv/math/problemgenerateor/service/EncryptionServiceTest.java
@@ -20,12 +20,12 @@ public class EncryptionServiceTest extends ProblemGenerateorApplicationTests {
         encryptionService.setKeyStr(GOOD_PASS);
         String encrypted = encryptionService.encrypt(text);
         Assert.assertNotNull("should have an encrypted message", encrypted);
-        Assert.assertEquals("Needs to match String", "7E89B6D66B7B001C7AE6955F122530D9", encrypted);
+        Assert.assertEquals("Needs to match String", text, encryptionService.decrypt(encrypted));
     }
 
     @Test
     public void decrypt() throws Exception {
-        String encrypted = "7E89B6D66B7B001C7AE6955F122530D9";
+        String encrypted = encryptionService.encrypt("Hello World!");
         encryptionService.setKeyStr(GOOD_PASS);
         String text = encryptionService.decrypt(encrypted);
         Assert.assertNotNull("should have an encrypted message", text);

